plugin/pkg/admission/serviceplan/changevalidator/admission.go

```diff
@@ -75,7 +76,7 @@ func (d *denyPlanChangeIfNotUpdatable) Admit(a admission.Attributes) error {
 	if err != nil {
 		if apierrors.IsNotFound(err) {
 			glog.V(5).Infof("Could not locate service class %v, can not determine if UpdateablePlan.", instance.Spec.ServiceClassName)
-			return nil
+			return nil // should this be `return err`? why would we allow the instance in if we cannot determine it is updatable?
```

Comment:
If memory serves me, I tried this with returning an error, it broke unit tests, so I figured this was the desired behavior.

Reply:
I suppose this particular thing is not concerned with whether the instance refers to a valid service class. We can let it through on the recognition that it will be caught later in the actual validation stages.

Weird, but I think that is the correct analysis. Thoughts?